### DISCUSSION TITLE
PNP search box without using enter

### DISCUSSION THREAD

**Discussion by AxelleHillewaereSavaco** [microsoft-search/pnp-modern-search#2713] [Ideas]
Is it possible to start the search part when I type in some characters in the search box? At the moment it's necessary to press enter or select the arrow to start the search but ideally my search starts without a manual action of the user.

![image](https://user-images.githubusercontent.com/122978039/214555018-b391db4f-23c3-4e38-b431-6c450dae1e58.png)

**Comment by kasperbolarsen** [comment]
I understand your point, but I guess the latency would often make this very unsatisfactory.  Perhaps search suggestions could provide a better solution?

**Reply by wobba** [reply]
To add, if you want execution per keypress for search results you need to either change the code or develop a custom search box.  As I don't think we want to add this capability to the project. Showing suggestions is likely a better UX approach.
